tests/mock_tables/dbconnector.py

```diff
@@ -34,6 +76,9 @@ class SwssSyncClient(mockredis.MockRedis):
     def __init__(self, *args, **kwargs):
         super(SwssSyncClient, self).__init__(strict=True, *args, **kwargs)
         db = kwargs.pop('db')
+        #namespace is added in kwargs specifically for unit-test
```

Comment:
>#na [](start = 8, length = 3)

Follow the comment convention with existing code:
1. Add a blank after `#`
2. Capitalize

Reply:
Fixed as per comment.